locales/it/index.yml

```diff
@@ -585,10 +586,10 @@ authentication:
     card2-content: Puoi pagare un tributo o una multa con il tuo conto corrente e le tue carte
     card3-title: Paga velocemente un avviso cartaceo
     card3-content: Tramite la scansione del QR code puoi velocemente pagare anche un avviso cartaceo
-    card4-title: Puoi accedere solo con SPID
+    card4-title: Puoi accedere solo con SPI D
```

Comment:
Is this a typo?

Reply:
Yep, thank you. BTW that card 's has been removed. So I guess we can delete those translations key too. What do you think?